examples/examples/form_examples.py

```diff
@@ -298,6 +302,45 @@ class Meta:
     return MyForm()
 
 
+@example(gettext("Nested forms -- to abstract out behaviour and create complex 'fields'."))
+def form_example_nested_forms(request):
+    """Here we have two fields first_day, last_day and want to abstract
+       out the validation behaviour.  Maybe because we have several
+       different forms that that edit different models that all have
+       a first_day and a last_day field."""
+    class DateRangeField(Form):
+        first_day = Field.date()
+        last_day = Field.date()
+
+        class Meta:
+            @staticmethod
+            def post_validation(form, **_):
+                print("post validation", form.is_valid())
+                if form.is_valid():
+                    if form.fields.first_day.value > form.fields.last_day.value:
+                        form.add_error("First day must be <= last day")
+
+    class MyForm(Form):
+        event = Field()
+        # attr='' => instance.first_day, instance.last_day instead of instance.when.first_day, instance.when.last_day
+        when = DateRangeField(attr='')
```

Comment:
I didn't understand this comment or why `attr=''` is here..

Reply:
Ok, now I got it.. but since I didn't get it the first time, maybe we can come up with something better...

Reply:
Yeah I wasn't super happy with that either.  Maybe a default of attr='' on Forms would be more correct...